File: trail1/20대의 평균 나이/average-age-of-20.c
#include <stdio.h>

int main() {
    int cnt = 0;
    int ssum = 0;
    while (1) {
        int n;
        scanf("%d", &n);
        if (n <20 || n>=30) {printf("%.2lf", (double)ssum / cnt); break;}
        ssum += n;
        cnt += 1;
    }
    return 0;
}

File: trail1/20대의 평균 나이/average-age-of-20.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cnt = 0;
        int ssum = 0;
        while(true) {
            int n = sc.nextInt();
            if (n >= 30 || n < 20) {System.out.printf("%.2f", (double)ssum / cnt); break;}
            ssum += n;
            cnt += 1;
        }

    }
}